README.md

```diff
@@ -156,6 +156,8 @@ The image can be pulled using :
 ```shell script
 docker pull quay.io/testnetworkfunction/test-network-function
 ```
+### Cluster requirement
+To run all the tests, OCP cluster should provide at least 3 worker nodes. If that's not the case, then ``platform-alteration`` test suite should be skipped.
```

Comment:
you mean scaling test?

Reply:
scaling and node draining.
and the comments exactly should be : 
you need to have (n+1) worker nodes if (n) is the maximum number of any CNF.

Reply:
any replica set?

Reply:
Yes. so if you have 10 replicas of vendor_cmm; because we have anti-Affinity, then each one will be in different worker. 
if we drain one worker, we will need one with no vendor_cmm in it. hence, we need 11 worker nodes.

But, we should also question this anti-affinity best practices (If I have 10 replicas and 4 workers), then I'm fine with having a distribution like (3-3-3-1) or (3-3-2-2), our test is rigid right now.

Reply:
basically, you take all the deployments, then figure out what's the maximum replicas in all of them. That would be your (n)

Reply:
Shouldn't we skip just the scaling and node draining test in platform alteration, since we have the new skip functionality?

Reply:
fixed.